package.json

```diff
@@ -43,33 +45,36 @@
   },
   "homepage": "https://github.com/galooshi/import-js#readme",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^8.0.0",
-    "babel-jest": "^21.0.2",
+    "@babel/cli": "^7.0.0-beta.55",
```

Comment:
It looks like beta 56 was published last week. Mind updating again?

Reply:
The semver range will include it and any later beta releases

Reply:
I still got beta.55 when I installed. Are you sure semver ranges work with pre-releases like these? Nothing to worry about though, I'm going to bump us to rc.1 anyway. :)

Reply:
It should, it should actually also include release candidates.